web/client/libs/ajax.js

```diff
@@ -31,6 +32,8 @@ function addHeaderToAxiosConfig(axiosConfig, headerName, headerValue) {
     axiosConfig.headers = assign({}, axiosConfig.headers, {[headerName]: headerValue});
 }
 
+const corsDisabled = [];
```

Comment:
I'm afraid this is static. Maybe you meant to make it configurable like `proxyConfig.useCORS` array ?

Reply:
No, it's not configurable, it records urls that do not support CORS so the test is not repeated and proxy is used for those.

Reply:
ok, sorry, I didn't get it. So... fine. No problem :smile: